pkg/cloudprovider/providers/openstack/openstack_instances.go

```diff
@@ -139,7 +153,13 @@ func (i *Instances) InstanceExistsByProviderID(ctx context.Context, providerID s
 	return true, nil
 }
 
-// InstanceShutdownByProviderID returns true if the instances is in safe state to detach volumes
+// InstanceShutdown returns true if the instance is shutdown.
+// It is the only state, where volumes can be detached immediately .
```

Comment:
maybe sync the comment for `InstanceShutdown ` and `InstanceShutdownByProviderID `